src/middleware/request-logging.ts

```
@@ -0,0 +1,640 @@
+  'x-session-id',
+  'x-csrf-token',
+  'x-xsrf-token',
+  'authentication',
+  'proxy-authorization',
+]
+
+// Log injection patterns
+const LOG_INJECTION_PATTERNS = {
+  // ANSI escape codes
+  // eslint-disable-next-line no-control-regex
+  ANSI_ESCAPE: /\x1b\[[0-9;]*[a-zA-Z]/g,
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)